.github/workflows/tests-ci.yml

```
@@ -64,8 +55,11 @@ jobs:
           node node_modules/@web5/dwn-server/dist/esm/src/main.js &
           echo "DWN_SERVER_BACKGROUND_PROCESS=$!" >> $GITHUB_ENV
 
+      - name: Build tests for all packages
+        run: pnpm --recursive --stream --sequential build:tests:node
+
       - name: Run tests for all packages
-        run: pnpm --recursive --no-stream --sequential test:node -- --color --reporter mocha-junit-reporter --reporter-options mochaFile=./results.xml
+        run: npm run test:node --ws -- --color
```

Revert to Tim's changes now that timeouts are resolved
```suggestion
        run: pnpm --recursive --stream exec c8 mocha -- --color --reporter mocha-junit-reporter --reporter-options mochaFile=./results.xml
```